docs/project/list-of-obsoletions.md

```diff
@@ -12,6 +12,7 @@ Currently the identifiers `MSLIB0001` through `MSLIB0999` are carved out for obs
 ## Current obsoletions (`MSLIB0001` - `MSLIB0999`)
 
 | Diagnostic ID    | Description |
-| :--------------- | :---------- |
-|  __`MSLIB0001`__ | The UTF-7 encoding is insecure and should not be used. Consider using UTF-8 instead. |
-|  __`MSLIB0002`__ | `PrincipalPermissionAttribute` is not honored by the runtime and must not be used. |
+| :-------------- | :-------------------------------------------------------------------------------------------------------- |
+| **`MSLIB0001`** | The UTF-7 encoding is insecure and should not be used. Consider using UTF-8 instead.                      |
+| **`MSLIB0002`** | `PrincipalPermissionAttribute` is not honored by the runtime and must not be used.                        |
+| **`TODO`**      | `CodeBase` / `EscapedCodeBase` is only included for .NET Framework compatibility. Use `Location` instead. |
```

Comment:
Needs a [Better Obsoletions](https://github.com/dotnet/designs/blob/master/accepted/2020/better-obsoletion/better-obsoletion.md) ID

Reply:
It'll be `SYSLIB0012`. See https://github.com/dotnet/designs/pull/139 for more info if desired. And note that this PR will need to be rebased on top of https://github.com/dotnet/runtime/pull/39269 once that is merged.

Reply:
Updated to use `SYSLIB0012`